config.example.wikimedia.yaml

```diff
@@ -169,6 +181,102 @@ templates:
 #              service: 
 #                uri: http://revscore.wikimedia.org/{domain}/{title}/{revision}
 
+  global-sys: &gb/sys/1.0.0
+    info:
+      title: global domain sys API module
+      version: 1.0.0
+    paths:
+      /{module:table}: *wp/sys/table
+      /{module:key_value}: *wp/sys/key_value
+      /{module:post_data}: *wp/sys/post_data
+      /{module:mathoid}/v1/request:
+        post:
+          x-request-handler:
+            - store:
+                request:
+                  method: put
+                  uri: /{domain}/sys/post_data/mathoid.request/
+                  body:
+                    q: '{$.request.body.q}'
```

Comment:
I must admit I like `formula` much better than `q`, but left it for compatibility.